source/extensions/filters/http/bandwidth_limit/bandwidth_limit.h

```diff
@@ -142,6 +148,8 @@ class BandwidthLimiter : public Http::StreamFilter, Logger::Loggable<Logger::Id:
   std::unique_ptr<Envoy::Extensions::HttpFilters::Common::StreamRateLimiter> response_limiter_;
   Stats::TimespanPtr request_latency_;
   Stats::TimespanPtr response_latency_;
+  uint64_t request_duration_ = 0;
```

Comment:
Would you want to use `std::chrono::milliseconds` to make units explicit?

Reply:
the source of request_duration_ is uint64, and this variable will be write to response trailer as string, use milliseconds will need additional convert from unit64 to milliseconds, then to string.